src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -137,9 +137,7 @@ export const MessagingThreadListItem: React.FC<
             </FlexBox>
           )}
         </FlexItem>
-        <FlexItem alignItems={ItemAlign.CENTER}>
-          {!isIndicatorAlignedWithSubContent && indicator}
-        </FlexItem>
+        <FlexItem>{!isIndicatorAlignedWithSubContent && indicator}</FlexItem>
```

Comment:
Wait I'm trying to understand this, is `alignItems` a prop for our `FlexItem` component, but not actually supported by browsers?

Reply:
There is no prop for alignItems on FlexItem! This was just a simple mistake of an additional prop that did nothing.

Reply:
Ahh I see, I thought TypeScript would have caught it then but we didn't type it enough https://github.com/Clever/components/blob/master/src/flex/FlexItem.tsx#L17-L18